### PR TITLE
Update bylaws.md

### PR DESCRIPTION
**Pull request by ftyers**
Some changes to the wording for grammaticality, style, and clarity.

### CODE REVIEW

File: bylaws.md [hunk 1]
```diff
@@ -54,20 +54,20 @@ v 2021-05-22
...
    counted when determining the majority.
 
-6. One vote per member of the organ the meeting is held for applies.
+6. For each meeting each member of the given organ has one vote.
 
 7. When electing persons, the following additional points apply:
 
-   - Each person who is elected must have own majority, meaning more than
-     half the votes must go to that person.
+   - Each person who is elected must have a majority, meaning that more than
```
Comment (swantzter) [inline bylaws.md:65]: "own majority" has a meaning as a legal term here

Comment (ftyers) [reply bylaws.md:65]: Not in English :) Do you mean "have their own majority" ? What is the term in Swedish ? *egen majoritet* ? Does it mean "simple majority"?

Comment (ftyers) [reply bylaws.md:65]: Here is the response I got from a colleague:

> ‘egen majoritet’ er når eit parti har absolutt majoritet sjølve, dvs utan å vera avhengig av støtteparti eller liknande - majoritet i kraft av sine eigne folk :)
> ‘enkel majoritet’ er den vanlege måten å ta avgjerder på, dvs ein ser bort frå fråverande eller blanke røyster, og ser berre på avlagde røyster: får eit forslag meir enn 50 % av røystene er det enkel majoritet.

Maybe the translation should be "overall majority", "their own overall majority" or "an absolute majority" ?

Comment (swantzter) [reply bylaws.md:65]: That could be a better translation.
Essentially in most other votes the MGM takes there is only yes or no, but for elections of people there may be multiple candidates. The winner isn't just the candidate with the most votes, but that candidate must have the most votes and more than 50% of the votes

Comment (ftyers) [reply bylaws.md:65]: I think it is probably better to spell out those conditions, e.g. "To be elected, each candidate must have (a) the most votes over all and (b) over 50% of the votes cast". I'll make a change and you can see what it looks like.

Comment (swantzter) [reply bylaws.md]: Not sure you need to spell out both really. If one candidate has more than 50% of the votes that automatically means they have "the most votes over all".

**Comment by swantzter** [comment]
closed in 961f4a67b4a4f9da5f10d24e9ca71fa93db1a81b

**Pull request closed without merging**